run-cnf-suites.sh

```diff
@@ -59,6 +59,14 @@ while [[ $1 == -* ]]; do
 	shift
 done
 
+# Check if the test cluster runs OCP
+res=`oc version | grep  Server`
+if [ -z "$res" ]
+then
+   echo "Minikube or similar detected"
```

Comment:
As part of this PR, do you want to change `Minikube` to `Kind` since we don't use minikube anymore.

Reply:
I'll rewrite it in a more generic way.